frontend/package.json

```diff
@@ -22,6 +22,8 @@
     "@types/testing-library__jest-dom": "^5.14.5",
     "@uswds/uswds": "^3.8.0",
     "apollo-upload-client": "^17.0.0",
+    "axios": "^1.7.2",
+    "babel-jest": "^29.7.0",
```

Comment:
nit: can we move this under the `devDependencies`?

Reply:
Change made